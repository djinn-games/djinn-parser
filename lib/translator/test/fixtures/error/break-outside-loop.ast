{
  "type": "Program",
  "name": {
    "type": "Identifier",
    "name": "break_outside_loop",
    "line": 1
  },
  "body": {
    "type": "Body",
    "sentences": [
      {
        "type": "LoopSentence",
        "sentences": [],
        "line": 3
      },
      {
        "type": "BreakSentence",
        "line": 5
      }
    ],
    "line": 2
  },
  "line": 1
}