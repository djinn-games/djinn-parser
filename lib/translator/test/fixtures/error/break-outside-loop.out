break sentence must be inside
